incubator/elk/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+description: A Helm chart for ELK
+name: elk
```

Comment:
I'd probably change the name to either:
```yaml
name: elk-stack
```
or 
```yaml
name: elastic-stack
```
See [An Introduction to the ELK Stack (Now the Elastic Stack)](https://www.elastic.co/webinars/introduction-elk-stack)